install_files/ansible-base/roles/ossec/tasks/configure_server.yml

```diff
@@ -113,3 +113,20 @@
     creates: /var/ossec/etc/sslmanager.cert
   tags:
     - ossec_auth
+
+- name: Copy the systemd service file
+  copy:
+    src: ossec.service
+    dest: "/etc/systemd/system/ossec.service"
+
+- name: Remove the old style /etc/init.d/ossec file
+  file:
+    path: "/etc/init.d/ossec"
```

Comment:
These files are provided by ossec-{agent,server} packages:

```
.
├── etc
│   ├── init.d
│   │   └── ossec
│   └── ossec-init.conf

```

This means that on subsequent ossec package upgrades, this init file will be re-added, unless we do specific tasks at build-time to remove it. If these steps are strictly required for functioning, we should address this in the build logic  as to not provide the init.d file in focal (instead of doing it in ansible)

Reply:
> This means that on subsequent ossec package upgrades, this init file will be re-added, unless we do specific tasks at build-time to remove it. If these steps are strictly required for functioning, we should address this in the build logic as to not provide the init.d file in focal (instead of doing it in ansible)

I think we should remove it from the server package.

Reply:
Definitely agree, let's remove the Ansible task and clean up that file at the packaging level. Be mindful of side-effects for Xenial.